cut_copy_paste_tests.py

```diff
@@ -279,11 +279,11 @@ def skip_comments():
             # Prepare scratch buffer
             buffer.run_command("select_all")
             buffer.run_command("right_delete")
-            buffer.run_command("append", { "characters": "".join([c[0] + '. ' + c[1] + "\n" for c in commands]) } ) # "insert" is not working totally correctly here, so "append" is used instead
-
-            # Process commands
-            for command in commands:
-                cmd = re.sub(R' \[[^]]+]', '', command[1]) # remove comments
+            buffer.run_command("append", { "characters": "".join([c[0] + '. ' + c[1] + "\n" for c in commands]) } ) # || "insert" is not working totally correctly here, so "append" is used instead
+                                                                                                                    # \\ To see what is the difference try

```

Comment:
Can you put these comments on their own lines instead of at the end of other lines?

Reply:
I don't totally understand you (do you mean place all 5 comment lines before `buffer.run_command("append", ...` or something else), but you can move this comments how you like. [Personally I see no problem with this comments because wide monitors is so widespread nowadays.]